plugins/fetch-yaml.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: fetch-yaml
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/olegsu/kubectl-fetch-yaml
+  shortDescription: "Fetch Kubernetes files from remote destinations"
+  description: |
+    Fetch Kubernetes files from remote destinations
+    Example git: kubectl fetch-yaml git --repo https://github.com/kubernetes/examples --path guestbook/all-in-one | kubectl applay -f -
+  platforms:
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+      uri: https://github.com/olegsu/kubectl-fetch-yaml/releases/download/v0.1.0/kubectl-fetch-yaml_0.1.0_Darwin_x86_64.tar.gz
+      sha256: 39a6b5e97a57b2a0e9bcb883392fd13c29632bfb78ea440ad659a46356cfb234
+      files:
```

Comment:
This is the default value. Remove “files:”

Reply:
Removed, thanks